src/components/interpolatedColorLegend.ts

```diff
@@ -55,15 +55,13 @@ export module Components {
     }
 
     /**
-     * Gets the current formatter on the InterpolatedColorLegend.
-     *
-     * @returns {Formatter} The current Formatter.
+     * Gets the current Formatter for the labels.
```

Comment:
Maybe change description to
"Gets the Formatter.
The Formatter is used to display labels."